docs/source/howto/installation.rst

```diff
@@ -558,13 +558,13 @@ In order to restore a backup, you will need to:
 
 Managing multiple users
 =======================
-Setups with multiple users for a single AiiDA instance are currently not supported.
-Instead, each AiiDA user should install AiiDA in a Unix/Windows account on their own computer.
-Under this account it is possible to configure all the credentials necessary to connect to remote computers.
-Using independent accounts will ensure that, for instance, the SSH credentials to connect to supercomputers are not shared with others.
+AiiDA currently does not support multiple users running concurrently on the same AiiDA instance (database & file repository).
```

Comment:
"AiiDA instance" can be a bit confusing here I think, because it could also be interpreted to mean the _installation_ itself. In that sense, an AiiDA instance can have one or more profiles that each configure a data store (database and file repository). What is not supported is to have multiple system users run simultaneously on the same _profile_ (or two different profiles that configure the same data store). It should, however, be just fine to have two different system users configure two different profiles in the same instance and run against those concurrently.

Reply:
Good point, will fix